test/test_framework_test.go

```diff
@@ -3889,36 +4249,29 @@ func TestGetEventsFromIntegrationTests(t *testing.T) {
 
 	const testCode = `
         import Test
-        import FooContract from 0x0000000000000005
+        import FooContract from "../contracts/FooContract.cdc"
 
-        pub let blockchain = Test.newEmulatorBlockchain()
-        pub let account = blockchain.createAccount()
+        pub let account = Test.getAccount(0x0000000000000005)
 
         pub fun setup() {
-            let contractCode = Test.readFile("../contracts/FooContract.cdc")
-            let err = blockchain.deployContract(
+            let err = Test.deployContract(
                 name: "FooContract",
-                code: contractCode,
-                account: account,
+                path: "../contracts/FooContract.cdc",
                 arguments: []
             )
 
             Test.expect(err, Test.beNil())
-
-            blockchain.useConfiguration(Test.Configuration({
-                "../contracts/FooContract.cdc": account.address
-            }))
```

Comment:
Great to see the changes help reduce all the boilerplate codes users had to write 👌